zulip_bots/zulip_bots/bots/mention/mention.py

```diff
@@ -0,0 +1,99 @@
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        message['content'] = message['content'].strip()
+
+        if message['content'].lower() == 'help':
+            bot_handler.send_reply(message, self.usage())
+            return
+
+        if message['content'] == '':
+            bot_handler.send_reply(message, 'Empty Mention Query')
+            return
+
+        keyword = message['content']
+        content = self.generate_response(keyword)
+        bot_handler.send_reply(message, content)
+
+    def get_account_id(self) -> str:
+        get_ac_id_header = {
+            'Authorization': 'Bearer ' + self.access_token,
+            'Accept-Version': '1.15',
+        }
+        response = requests.get(
+            'https://api.mention.net/api/accounts/me', headers=get_ac_id_header)
+        response = response.json()
+        account_id = response['account']['id']
+        return account_id
+
+    def get_alert_id(self, keyword: str) -> str:
+        create_alert_header = {
+            'Authorization': 'Bearer ' + self.access_token,
+            'Content-Type': 'application/json',
+            'Accept-Version': '1.15',
+        }
+        create_alert_data = '{"name":"' + keyword + \
+            '","query": {"type": "basic","included_keywords":["' + \
+            keyword + '"]},"languages":["en"],"sources":["web"]}'
+        response = requests.post('https://api.mention.net/api/accounts/' + self.account_id +
+                                 '/alerts', headers=create_alert_header, data=create_alert_data)
+        response = response.json()
+        alert_id = response['alert']['id']
+        return alert_id
+
+    def get_mentions(self, alert_id: str) -> List[Any]:
+        get_mentions_header = {
+            'Authorization': 'Bearer ' + self.access_token,
+            'Accept-Version': '1.15',
+        }
+        response = requests.get('https://api.mention.net/api/accounts/' + self.account_id +
+                                '/alerts/' + alert_id + '/mentions', headers=get_mentions_header)
+        response = response.json()
+        mentions = response['mentions']
+        return mentions
+
+    def generate_response(self, keyword: str) -> str:
+        if self.account_id == '':
+            self.account_id = self.get_account_id()
+
+        try:
+            alert_id = self.get_alert_id(keyword)
+        except (TypeError, KeyError):
+            # usually triggered by Invalid Token or json parse error when account quote is finished
+            raise MentionNoResponseException()
+
+        try:
+            mentions = self.get_mentions(alert_id)
+        except (TypeError, KeyError):
+            # usually triggered by No response or json parse error when account quota is finished
+            raise MentionNoResponseException()
+
+        reply = 'The most recent mentions of `' + keyword + '` on the web are: \n'
+        for mention in mentions:
+            reply += '[' + mention['title'] + \
```

Comment:
I suggest using the format method instead of string concatenation:
```
reply += "[{title}]({id})\n".format(title=mention['title'], id=mention['unique_id'])
```
I think it would be more readable.